modules/optimization/test/tests/optimizationreporter/general_opt/point_loads/main_seperate_multiapps.i

```diff
@@ -0,0 +1,114 @@
+measurement_points = '0.5 0.28 0
+   0.5 0.6 0
+   0.5 0.8 0
+   0.5 1.1 0'
+measurement_values = '293 304 315 320'
```

Comment:
Have to do it this way to allow me to change the points and values for the
regularized version.